### PR TITLE
initial blog post for carbon-route

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

**Comment by tjsavage** [comment]
cc: @rictic @arthurevans - what do you think?

File: _posts/2016-03-28-carbon-route-released.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+date:   2016-03-28 10:30:00
+categories: announcements
+author: polymer-team
+excerpt: "The `carbon-route` element has made its beta debut - we'd love your feedback on this modular approach to routing."
+
+---
+
+We're excited to announce the beta release of the [`carbon-route`](https://github.com/polymerelements/carbon-route) element - a composable, modular web component router! To dive into this new element and see what it can do, check out the [Encapsulated Routing with Elements](https://www.polymer-project.org/1.0/articles/routing.html) article on the Polymer site.
+
+The `carbon-route` element provides a declarative way to let web components hook in to the current URL, and route behavior accordingly. It is currently in its `v0.8` iteration - complete, but still may evolve based on feedback before it reaches `v1.0`. As an example of how the `carbon-route` element may be used:
```
Comment (rictic) [inline _posts/2016-03-28-carbon-route-released.markdown]: What do you think about this wording for the first sentence?

The `carbon-route` element provides a declarative, modular way for web components to interact with the URL.

File: _posts/2016-03-28-carbon-route-released.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                     exit-animation='slide-right-animation'>
+  <neon-animatable key='foo'>Foo Page Here</neon-animatable>
+  <neon-animatable key='bar'>Bar Page Goes Here</neon-animatable>
+  <neon-animatable key='baz'>Baz Page, the Best One of the Three</neon-animatable>
+</neon-animated-pages>
+```
+
+The `carbon-route` element is the first of the Polymer project's "carbon" product-line of components - a set designed to help encapsulate structured patterns for building large, complex applications using web components. Other elements that may fill out this product line - including i18n components - are still in their earlier stages. We'll being opening up these elements in a similar "beta" stage once they're at a point of stability to be able to receive meaningful feedback, and are targeting May to have the next handful of these components released. We'd also love to hear about the common application-level problems you run into when building web component-driven apps, that could form the basis of future carbon elements - please feel free to file issues on the [`carbon-elements` meta-repo](https://github.com/polymerelements/carbon-elements/issues) with ideas.
```
Comment (rictic) [inline _posts/2016-03-28-carbon-route-released.markdown]: I like this paragraph a lot.

**Comment by rictic** [comment]
This looks good to me. Do we want to wait for the elements catalog to be updated first?

**Comment by rictic** [comment]
I'm also working on making our demos more catalog-friendly. I think I can have this good to go by early this afternoon: https://github.com/PolymerElements/carbon-route/issues/53

File: _posts/2016-03-28-carbon-route-released.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+excerpt: "The `carbon-route` element has made its beta debut - we'd love your feedback on this modular approach to routing."
+
+---
+
+We're excited to announce the beta release of the [`carbon-route`](https://github.com/polymerelements/carbon-route) element - a composable, modular web component router! To dive into this new element and see what it can do, check out the [Encapsulated Routing with Elements](https://www.polymer-project.org/1.0/articles/routing.html) article on the Polymer site.
+
+The `carbon-route` element provides a declarative way to let web components hook in to the current URL, and route behavior accordingly. It is currently in its `v0.8` iteration - complete, but still may evolve based on feedback before it reaches `v1.0`. As an example of how the `carbon-route` element may be used:
+
+```
+<carbon-route pattern="/tabs/:tabName" data="{{data}}"></carbon-route>
```
Comment (abdonrd) [inline _posts/2016-03-28-carbon-route-released.markdown]: Here we need the `carbon-location`, right?

`<carbon-location route="{{route}}"></carbon-location>`

File: _posts/2016-03-28-carbon-route-released.markdown (hunk 1 above)
Comment (arthurevans) [inline _posts/2016-03-28-carbon-route-released.markdown]: I don't see any need to codefont the versions. I also think "beta" is more meaningful than "0.8", which can honestly mean almost anything... Perhaps:

"This beta version is complete but may still evolve based on your feedback."
"As an example ..." => Here's an example of how you can use the `carbon-route` element?

I also recommend:
s/ - /—/g

**Comment by arthurevans** [comment]
LGTM :+1:

File: _posts/2016-03-28-carbon-route-released.markdown (hunk 1 above)
Comment (tjsavage) [inline _posts/2016-03-28-carbon-route-released.markdown]: much better

**Comment by ebidel** [comment]
LGTM

**Comment by rictic** [comment]
It's live in the element catalog now. LGTM.

File: _posts/2016-03-28-carbon-route-released.markdown (hunk 3 above)
Comment (abdonrd) [inline _posts/2016-03-28-carbon-route-released.markdown]: Friendly ping to @rictic. :)

Comment (rictic) [inline _posts/2016-03-28-carbon-route-released.markdown]: Yeah, if we're gonna include this code as a full fledged example then yes, we should add carbon-location. This example is taken from the routing article, where it's just used to give the flavor for how carbon-route integrates with the other Polymer elements.

Comment (tjsavage) [inline _posts/2016-03-28-carbon-route-released.markdown]: Thanks @abdonrd! Completeness will be good, I'll make sure to include `carbon-location` as well.

**Pull request merged**